cve_bin_tool/checkers/gnomeshell.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+CVE checker for gnome-shell
+
+https://www.cvedetails.com/product/20202/?q=Gnome-shell
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class GnomeshellChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"\* Creates a window using gnome-shell-perf-helper for testing purposes."
+    ]
+    FILENAME_PATTERNS = [r"gnome-shell"]
+    VERSION_PATTERNS = [r"var PACKAGE_VERSION = '([0-9]+\.[0-9]+(\.[0-9]+)?)';"]
```

Comment:
My gut says this string is likely to generate false positives eventually.  Is there anything better available, and if not, can we at least try grepping all the gnome package strings (or maybe the strings for a wider range of linux packages) to make sure this isn't a config pattern used elsewhere?

Reply:
There are nearly 100 different packages listed while searching "gnome", of which plenty of them starts with `gnome-shell`.
And I think this could most probably hit false positives.

I looked through the strings in the package a bit and found the version strings are placed 
```
var PACKAGE_NAME = 'gnome-shell';
var PACKAGE_VERSION = '40.2';
```
but `VERSION_PATTERNS` doesn't have multi-line support atm.
So I'm wondering if I should close this or try tinkering with enabling multiline support.